### PR TITLE
Wes whitney

### PR DESCRIPTION
**Pull request by weswhitney**

### CODE REVIEW

File: write_ups/describe_these_code_examples.md [hunk 1]
```diff
@@ -5,10 +5,12 @@ Use terms like assigned, reassigned, statement, and concatenates to explain each
...
 
 ```
-x = 7 
+x = 7
 ```
 
-**Response:** 
+**Response:**
+
+statement
```
Comment (amk-boCO) [inline write_ups/describe_these_code_examples.md:13]: Looking for a bit more words on these answers, something like "The value `7` is assigned to the variable `x`" is concise, clear, and indicates exactly what is going on.

File: write_ups/describe_these_code_examples.md [hunk 2]
```diff
@@ -19,6 +21,7 @@ puts fooey
 ```
 
 **Response:**
+assigned
```
Comment (amk-boCO) [inline write_ups/describe_these_code_examples.md:24]: Same here

File: write_ups/describe_these_code_examples.md [hunk 3]
```diff
@@ -29,13 +32,15 @@ speical_number = 3
 ```
 
 **Response:**
+reassigned
```
Comment (amk-boCO) [inline write_ups/describe_these_code_examples.md:35]: Same here

File: write_ups/describe_these_code_examples.md [hunk 4]
```diff
@@ -29,13 +32,15 @@ speical_number = 3
...
 
 4)
 
 ```
 1 + 2
 ```
-**Response:** 
+**Response:**
+statement
```
Comment (amk-boCO) [inline write_ups/describe_these_code_examples.md:43]: Same here

File: write_ups/describe_these_code_examples.md [hunk 5]
```diff
@@ -47,7 +52,8 @@ a + b
 
 ```
 
-**Response:** 
+**Response:**
+concatenates
```
Comment (amk-boCO) [inline write_ups/describe_these_code_examples.md:56]: Same here

File: write_ups/describe_these_code_examples.md [hunk 6]
```diff
@@ -56,4 +62,5 @@ z = 9
 z = z * 3
 ```
 
-**Response:** 
\ No newline at end of file
+**Response:**
+reassigned
```
Comment (amk-boCO) [inline write_ups/describe_these_code_examples.md:66]: Same here

File: write_ups/write_code_to_match_the_explanation.md [hunk 7]
```diff
@@ -4,66 +4,85 @@ Write code to match each of the following explanations.
...
 ```
 
 4) Provide a statement that concatenates the values `6` and `8`
 
-**Response:** 
+**Response:**
 
 ```
-# your code here
+"6" + "8"
```
Comment (amk-boCO) [inline write_ups/write_code_to_match_the_explanation.md:36]: This was actually a trick question! You don't concatenate Integers!

File: write_ups/write_code_to_match_the_explanation.md [hunk 8]
```diff
@@ -4,66 +4,85 @@ Write code to match each of the following explanations.
...
 #### Revisiting Classes and Objects
 
 6) Evaluate the final statement from *Question #5*.
 
-**Response:** 
+**Response:**
 
 ```
-# your code here
+"231313 arbitrary"
```
Comment (amk-boCO) [inline write_ups/write_code_to_match_the_explanation.md:56]: Are you sure this is what the output would look like? Not sure if I agree :)

File: write_ups/write_code_to_match_the_explanation.md [hunk 9]
```diff
@@ -4,66 +4,85 @@ Write code to match each of the following explanations.
...
+
+```
+class Car
+  def initialize(make, model)
+    @make = make
+    @model = model
+  end
+
+  def honk
+    puts "#{@make.upcase} + goes BEEP!"
```
Comment (amk-boCO) [inline write_ups/write_code_to_match_the_explanation.md:84]: Will this output `"SUBARU goes BEEP!"` or `"SUBARU + goes BEEP!"`?

**Comment by weswhitney** [comment]
cleaning up my PR section in my profile.

**Pull request closed without merging**